### PR TITLE
Office Hours 2019-10-15

### PR DESCRIPTION
**Pull request by sharvilshah**
This is of course a work in progress, and lacks polish. But I think this covers most of the things discussed in OH.

### CODE REVIEW

**Comment by theopolis** [comment]
@directionless, what do you think?

**Comment by theopolis** [comment]
@directionless, what do you think?

File: docs/office-hours/20191015_meeting_notes.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+We need Apple account for osquery. These will be a required with notarizing macOS packages going forward.
+
+(Sharvil); Able to notarize a simple hello world binary w/ his personal account. Unsure yet how to go about with pkgs. This needs investigating.
+
+Seph via slack: The Linux foundation has said they don't have an ADC account. It's on us to run that riddle trail. And either pay, or get the tax id for LF
+
+## `auditd` vs `audisp`
+
+Issue #5594.
```
Comment (directionless) [inline docs/office-hours/20191015_meeting_notes.md]:
```suggestion
Issue https://github.com/osquery/osquery/issues/5594
```

File: docs/office-hours/20191015_meeting_notes.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+## Outreach
+
+### Code Review
+
+* Ted/Zach: it would be nice to have people help out with code reviews. 
+* Github notifications are not helpful. It would be nice to have a Slack channel where people can request review.
+* Zach: We now have `#code-review` channel in Slack. We should update PR template to mention a blurb about it.
+* Ted: We should have more committers. General consensus is that we should reach out to involved community members with previous contributions.
+
```
Comment (directionless) [inline docs/office-hours/20191015_meeting_notes.md]:
```suggestion
* See also https://github.com/osquery/foundation/issues/36

```

File: docs/office-hours/20191015_meeting_notes.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Documentation: https://wiki.openssl.org/index.php/Compilation_and_Installation#Modifying_Build_Settings
+
+* Sharvil: we might need to install full Xcode (not just command line tools) to get the sysroot and full 10.15 SDK, we are unsure, we may have to install another pkg on Azure and update our docs.
+* Lack of Catalina stability and access to machine could be blocker.
+* Some osquery tables are slow on Catalina (Sharvil is investigating).
+
+## Apple Developer Account
+
+We need Apple account for osquery. These will be a required with notarizing macOS packages going forward.
```
Comment (directionless) [inline docs/office-hours/20191015_meeting_notes.md]:
```suggestion
We need Apple account for osquery. These will be a required with notarizing macOS packages going forward. (https://github.com/osquery/foundation/issues/3)
```

File: docs/office-hours/20191015_meeting_notes.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+## OpenSSL and macOS Catalina
+
+We need to switch Windows and macoS to use CMake/formula version. 
+
+* Sharvil: Upgrading to 10.15 SDK has issues.
+* Sharvil: On 10.15 there doesn't seem to be a way to have `/usr/include`.
+* Alessandro: building OpenSSL with CMake is involved, following patch helps.
+
+```cmake
```
[the remark this answers is not in the document]
Comment (sharvilshah) [reply docs/office-hours/20191015_meeting_notes.md]: IIRC this patch might be from OpenSSL wiki/docs.

File: docs/office-hours/20191015_meeting_notes.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+Groob asks if Ryan will be engaged again. We learn that this may not be the best meeting time for Ryan. Ted will try to find out.
+
+Ted is merging bug fixes from GH to FB. It's been pretty smooth. One recent issue about http tests was created due to IPv6 testing. (https://github.com/osquery/osquery/issues/5883)
+Need to revist `http_client` and changes in (PR #5606) to check IPv6 compatibility.
+
+## Extensions support
+
+* (PR #5851) is ready
+* A small number of previously exported functions are no longer available as they were removed from the codebase. If they are no longer used internally, we should treat as dead code and we can remove them. In particular, this is in reference to two functions for serializing/deserializing json -- consensus is that they should be implemented in extensions if needed
```
Comment (directionless) [inline docs/office-hours/20191015_meeting_notes.md]:
```suggestion
* A small number of previously exported functions are no longer available as they were removed from the codebase. If they are no longer used internally, we should treat as dead code and we can remove them. In particular, this is in reference to two functions for serializing/deserializing json -- consensus is that they should be implemented in extensions if needed. (https://github.com/osquery/osquery/issues/5878)
```

**Pull request merged**